OpenRA.Game/Game.cs

```diff
@@ -619,7 +619,8 @@ static void InnerLogicTick(OrderManager orderManager)
 					{
 						++orderManager.LocalFrameNumber;
 
-						Log.Write("debug", "--Tick: {0} ({1})", LocalTick, isNetTick ? "net" : "local");
+						if (Settings.Debug.EnableDebugTickLogging)
+							Log.Write("debug", "--Tick: {0} ({1})", LocalTick, isNetTick ? "net" : "local");
```

Comment:
IMO we should just delete this entirely.

Reply:
On second thought, this is actually an interesting number to extract and plot so how about we move this to perf.log and the perf debug setting?

Reply:
On third thought, this is just logging the tick numbers without any performance information. Ignore my last comment and lets just delete this!